pages/blog/conference-2023.md

```diff
@@ -0,0 +1,168 @@
+
+## Challenges
+
+Our biggest challenge of all was that the AACoT'23 planning started late. This meant that for every decision and effort we made, we were dashing against the clock.
+
+- Planning and marketing were complex since most venues were confirmed a few weeks before the summer holidays.
+- In terms of promoting the events, we had some challenges in certain locations as we expected some support in this regard. This would have helped boost the event's visibility, but we were partly at fault since this wasn't communicated prior.
+- Sometimes, free things can be costly! All the conference tickets were free. Of as many those who redeemed free tickets through Open Collective, only less than 50% showed up. It was very discouraging as food and resources that could have catered for everyone went to waste.
+- There was no time to rest between the events as they all happened one after the other.
+- We didn't have a sponsorship budget. By the time we thought about this, it was way too late. 
+
+## Lessons
+
+Time and money are essential! When it comes to building a brand and conference planning.
+I can't over-emphasize the importance of early planning, as we have seen in the past year, which we will prioritize for AACoT'24.
+
+Firstly, having open communication and fostering strong partnerships with our sponsors is undeniable. Before we agree and confirm the location, each venue must meet our community's minimum requirements, especially regarding support in marketing and participation in the planning process.
+
+Secondly, open up sponsorship tiers for companies and individuals who want to support us. Running AACoT requires a lot of funds, especially when it comes to swag costs, shipping, and going further to provide travel scholarships to community members who would like to participate. So, having a separate budget for AACoT is very crucial in this regard.
+
+Lastly, spacing all the conferences this year. Instead of what happened in 2023, where we had all meetups lined up a few weeks apart from each other, in 2024, we plan to have one conference each quarter. Additionally, this will give people time to prepare and plan to travel.
+
+##  Attendance
+
+Though you may have an idea of how the attendance went, the best takeaway is that we had time to network, connect, and learn during the conferences. 
+Being our first year, we did fairly well regarding attendance numbers. Here is a breakdown of attendance in each city: 
+- In London, over 30 people attended in person. 
+- In Madrid, we had 50 people attend in person, and the livestream had 437 views. 
+- In Bangalore, we had around 25 people attend in person, and the livestream received 377 views. 
+- In Paris, we don't have a total number, but we had a full house on the conference day. We had a booth where we advertised AsyncAPI on a vendor-neutral stand.
```

Comment:
```suggestion

In total we hosted 225+ in person. Two events were livestreamed and it helped us to reach 800+ viewers.

Additionally, in Paris we did not only have an conference track on one day, but throughout the whole three days conference we also organized an `API Specs and Standards` booth with friends from OpenAPI, JSON Schema and GraphQL where we had countless discussions with many different people from API world.
```

this is a rough proposal how we can change this section to focus on total number that is the most important for potential sponsors

Reply:
Lol i forgot to push the changes